content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+  caption="Upgrade to 4.0 directly from version 3.20 or higher with multi-version upgrades"
+/>
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<a href="https://docs.sourcegraph.com/admin/analytics"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Understand usage and quantify the value of using Sourcegraph with in-product analytics
+Our new in-product analytics help admins to understand user engagement, measure efficiency in terms of time saved, and calculate the value of Sourcegraph and its features. 
+
+These analytics work across all of our products and provide ROI-based metrics such as:
+
+- The number of searches developers have run in Sourcegraph, and how many hours of development time have been saved as a result.
+- How many code insights have been created, viewed, and interacted with.
+- The number of changesets that have been created and merged via batch changes, and how much time has been saved as a result.
+- How frequently code navigation has been used by developers to more quickly understand code.
+- How often IDE and Browser extensions are used across your organization, and how much time has been saved as a result.
+- The number of navigation events and the percentage of your repositories that allow precise code navigation
+
+Admins can select a specific date range and customize the assumpted to align with how the platform is used. 
+
+<div className="tw-mb-sm">
+  <Figure
+    src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search.png"
+    alt="In-product analytics"
+    caption="Docs"
+    link="https://docs.sourcegraph.com/admin/analytics"
+    linkIcon={true}
+  />
+</div>
+
+<br />
+<a href="https://docs.sourcegraph.com/admin/observability/opentelemetry"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Easily export traces using OpenTelemetry
+
+[OpenTelemetry (OTEL)](https://opentelemetry.io/) is an open specification for exporting observability data into a user’s preferred systems. We aim to make OpenTelmetry the default way to quickly and easily export observability data from Sourcgraph, starting with tracing in Sourcegraph 4.0. This is a breaking change as we now require you to export tracing with the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). This is deployed by default in all Sourcegraph deployment methods and replaces the Jaeger agents and collectors that was previously the default. 
+
+<a href="https://docs.sourcegraph.com/admin/observability/opentelemetry" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<a href="https://docs.sourcegraph.com/admin/external_service"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Quickly see the status on your repository and permissions syncing
```

Comment:
@ryphil you please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.